app/torii-providers/arcgis-oauth-bearer.js

```diff
@@ -39,6 +42,36 @@
    _currentBaseUrl: function () {
      return [window.location.protocol, '//', window.location.host].join('');
    },
+
+   buildQueryString: function(options){
+     const requiredParams = this.get('requiredUrlParams');
+     const optionalParams = this.get('optionalUrlParams');
+
+     const qs = QueryString.create({
+       provider: this,
+       requiredParams: requiredParams,
+       optionalParams: optionalParams,
+       options
+     });
+
+     return qs.toString();
+  },
```

Comment:
This is the same as the torii default implementation except that it passes `options` along.